### PR TITLE
Update National Minimum Wage for past values when over 25

### PR DESCRIPTION
**Pull request by pmanrubia**
Trello card: https://trello.com/c/nV9KmLD1/90-fix-issue-national-living-wage-for-past-years-when-over-25

Addresses issue with #2423 as paths related to old years when user is over 25 are not honoring [the specification document with the required changes](https://docs.google.com/document/d/1HYr7oPtx1zyGo1uiq4-dlqljsfVzR7YKRcc0zhvRksw/edit)

# Factchecks
* [Minimum Wage Calculator Employers](https://smart-answers-pr-2439.herokuapp.com/minimum-wage-calculator-employers/)
* [Minimum Wage Calculator Employee](https://smart-answers-pr-2439.herokuapp.com/am-i-getting-minimum-wage/)

# Expected Changes

* [Minimum Wage Calculator Employers](https://gov.uk/minimum-wage-calculator-employers/)
* [Minimum Wage Calculator Employee](https://gov.uk/am-i-getting-minimum-wage/)

### Before
screen shot 2016-04-06 at 09 12 25

### After
screen shot 2016-04-06 at 09 12 30

### CODE REVIEW

File: test/unit/calculators/minimum_wage_calculator_test.rb [hunk 1]
```diff
@@ -161,6 +161,12 @@ class MinimumWageCalculatorTest < ActiveSupport::TestCase
           assert !@calculator.eligible_for_living_wage?
         end
       end
+
+      should 'return false if age is over 25, and date is before 2016-04-01' do
```
Comment (ikennaokpala) [inline test/unit/calculators/minimum_wage_calculator_test.rb]: Minor: The narrative here I feel should read thus:
```ruby should  'return false if age is over 25, and date is on or before 2016-04-01' do```

This is based on [line](https://github.com/alphagov/smart-answers/blob/national-minimum-wage-update/lib/smart_answer/calculators/minimum_wage_calculator.rb#L174) 174 in `lib/smart_answer/calculators/minimum_wage_calculator.rb`

File: test/unit/calculators/minimum_wage_calculator_test.rb [hunk 2]
```diff
@@ -161,6 +161,12 @@ class MinimumWageCalculatorTest < ActiveSupport::TestCase
...
+        @calculator.date = Date.parse('2016-03-30')
+        @calculator.age = 26
+        assert !@calculator.eligible_for_living_wage?
```
Comment (ikennaokpala) [inline test/unit/calculators/minimum_wage_calculator_test.rb]: Not sure about this `assert !@calculator.eligible_for_living_wage?` is this intentional ?

**Comment by ikennaokpala** [comment]
Apart from the test narratives [here](https://github.com/alphagov/smart-answers/pull/2439/commits/e33828abe493be4e599a27095f7f40aa9ffb25b8#r58678077).. This PR LGTM

**Comment by pmanrubia** [comment]
Hi @ikennaokpala thanks for your quick review. I have updated with your comments. Also rebased and pushed in preparation for merging.

**Comment by ikennaokpala** [comment]
@pmanrubia great use of `refute` reads better now

File: lib/data/minimum_wage_data.yml [hunk 3]
```diff
@@ -136,7 +136,10 @@
     :max_age: 21
     :rate: 5.30
   - :min_age: 21
-    :max_age: 1000
+    :max_age: 24
```
Comment (chrisroos) [inline lib/data/minimum_wage_data.yml]: The `MarriageAbroadCalculator#per_hour_minimum_wage` method uses `@age < r[:max_age]` which means this rate isn't picked up for people aged 24. I've created PR #2441 to fix this problem.

**Pull request merged**